### PR TITLE
Installation instruction updated to fit current progress

### LINKED ISSUE #36: Update installation instruction on README.md
Since our package is not publish on PyPI yet, we should mention this on our README too.

### PR DESCRIPTION
**Pull request by s-voon**
Hi guys, I added instruction to install our package through poetry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,26 @@ This package focuses on eight widely used regressor models, providing a curated
...
+
+### Initialization
+1. Clone this GitHub repository using this command:
+```bash
+git clone https://github.com/UBC-MDS/autopredictor.git
+```
+
+2. Install poetry in your base environment through this [instruction](https://python-poetry.org/docs/#installation).
+
+3. Run the following commands from the root directory of this project to create an virtual environment for this package and install autopredictor through poetry:
```
Comment (AReyH) [inline README.md]:
```suggestion
3. Run the following commands from the root directory of this project to create a virtual environment for this package and install autopredictor through poetry:
```

Corrected the "to create a virtual environment"

File: README.md [hunk 2]
```diff
@@ -12,6 +12,26 @@ This package focuses on eight widely used regressor models, providing a curated
 
 ## Installation
 
+Since this package is still in the developing process, it has not been published on PyPI yet. Thus, In order to use this package, please run the instructions provided below.
+
+### Initialization
+1. Clone this GitHub repository using this command:
```
Comment (zgarciaj) [inline README.md:18]: Optional:
We might want to add a note about where the command should be run > terminal or or command prompt/Git Bash.

File: README.md [hunk 3]
```diff
@@ -12,6 +12,26 @@ This package focuses on eight widely used regressor models, providing a curated
...
+```bash
+git clone https://github.com/UBC-MDS/autopredictor.git
+```
+
+2. Install poetry in your base environment through this [instruction](https://python-poetry.org/docs/#installation).
```
Comment (zgarciaj) [inline README.md]:
```suggestion
2. Install poetry in your base environment by following these [instructions](https://python-poetry.org/docs/#installation).
```

File: README.md (hunk 1 above)
Comment (zgarciaj) [inline README.md]:
```suggestion
3. Run the following commands from the root directory of this project to create a virtual environment for this package and install autopredictor through poetry:
```

**Review by AReyH — APPROVED** [review]
LGTM! I think it covers all bases.

**Pull request merged**